pkgs/servers/web-apps/pixelfed/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, phpPackages
+, pkgs
+, dataDir ? "/var/lib/pixelfed"
+}:
+
+let
+  package = (import ./composition.nix {
+    inherit pkgs;
+    inherit (stdenv.hostPlatform) system;
+    noDev = true; # Disable development dependencies
+  }).overrideAttrs (attrs : {
+    installPhase = attrs.installPhase + ''
+      rm -R $out/bootstrap/cache $out/storage
+      ln -s ${dataDir}/.env $out/.env
+      ln -s ${dataDir}/storage $out/
+      ln -s ${dataDir}/storage/app/public $out/public/storage
+      ln -s ${dataDir}/bootstrap/cache $out/bootstrap/cache
+      chmod +x $out/artisan
+    '';
+  });
+
+in package.override rec {
+  pname = "pixelfed";
+  version = "UNSTABLE-01-09-2022";
```

Comment:
This doesn't line up with the version in the commit message, and even if it did, it's incorrect (should be `unstable-2022-09-01`).

Reply:
yes, the PR is really very drafty now because I am doing something else, I was planning to use a proper tag version

Reply:
Fixed